src/aws_encryption_sdk_cli/internal/arg_parsing.py

```diff
@@ -85,22 +86,33 @@ def add_argument(self, *args, **kwargs):
 
         return super(CommentIgnoringArgumentParser, self).add_argument(*args, **kwargs)
 
+    def __parse_line(self, arg_line):
+        # type: (ARGPARSE_TEXT) -> List[str]
+        """Parses a line of arguments into individual arguments intelligently for different platforms.
+
+        :param str arg_line: Raw argument line
+        :returns: Parsed line members
+        :rtype: list of str
+        """
+        shlexer = shlex.shlex(six.StringIO(arg_line), posix=True)  # type: ignore #  shlex confuses mypy
+        shlexer.commenters = '#'
```

Comment:
Isn't `#` already the default value for this parameter? Why is this configuration (but not others) being set explicitly to its default?

Reply:
I think I set it just out of paranoia because it's something we specifically care about, but you're right; this has stayed the same default value since something like Python 2.6, I think it's reasonable to assume it will continue being the same value. Removing explicit set.